lib/hocon/impl/simple_config_list.rb

```diff
@@ -21,6 +24,8 @@ def initialize(origin, value, status = ResolveStatus.from_values(value))
     end
   end
 
+  def_delegators :@value, :[], :include?, :empty?, :size, :index, :rindex, :each, :map
+
```

Comment:
It's just a helper that defines methods for the class that take the methods listed (`[]`, `include?`, `size`, etc) and point them to the `@value` variable. In the java version the class implements the methods the long way 

```java
@Override
    public boolean contains(Object o) {
        return value.contains(o);
    }
```

...if that makes sense. It seemed to be the idiomatic ruby way of doing things